stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -36,6 +36,12 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
+% if CMAKE_SDK in ["FREEBSD"]:
+  module cdefs {
+    header "sys/cdefs.h"
+  }
```

Comment:
should we export everything from `cdefs.h` ?

Reply:
Yeah, missed that in the upload.